src/builtins_part_one.c

```diff
@@ -0,0 +1,173 @@
+// helloworld
+// 2-G-4:~ chuhlig$ echo ''
+
+// 2-G-4:~ chuhlig$ echo "$PWD"
+// /Users/chuhlig
+// 2-G-4:~ chuhlig$ echo '$PWD'
+// $PWD
+// 2-G-4:~ chuhlig$ echo "aspas ->'"
+// aspas ->'
+// 2-G-4:~ chuhlig$ echo "aspas -> ' "
+// aspas -> ' 
+// 2-G-4:~ chuhlig$ echo 'aspas ->"'
+// aspas ->"
+// 2-G-4:~ chuhlig$ echo 'aspas -> " '
+// aspas -> " 
+// 2-G-4:~ chuhlig$ echo "> >> < * ? [ ] | ; [ ] || && ( ) & # $ \ <<"
+// > >> < * ? [ ] | ; [ ] || && ( ) & # $ \ <<
+// 2-G-4:~ chuhlig$ echo '> >> < * ? [ ] | ; [ ] || && ( ) & # $ \ <<'
+// > >> < * ? [ ] | ; [ ] || && ( ) & # $ \ <<
+// 2-G-4:~ chuhlig$ echo "exit_code ->$? user ->$USER home -> $HOME"
+// exit_code ->0 user ->chuhlig home -> /Users/chuhlig
+// 2-G-4:~ chuhlig$ echo 'exit_code ->$? user ->$USER home -> $HOME'
+// exit_code ->$? user ->$USER home -> $HOME
+// 2-G-4:~ chuhlig$ echo "$"
+// $
+// 2-G-4:~ chuhlig$ echo '$'
+// $
+// 2-G-4:~ chuhlig$ echo $
+// $
+
+// 2-G-4:~ chuhlig$ echo "test" -n hello
+// test -n hello
+// 2-G-4:~ chuhlig$ echo -n "test" -n hello
+// test -n hello2-G-4:~ chuhlig$ echo -n "test" echo -n hello
+// test echo -n hello2-G-4:~ chuhlig$ 
+
+
+
+// 2-G-4:~ chuhlig$ echo -n "test1 test2" test3
+// test1 test2 test32-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n " test1 test2 " 'test3 '
+//  test1 test2  test3 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n test1test2
+// test1test22-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo test1 -n
+// test1 -n
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo "test1 -n"
+// test1 -n
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -n test1
+// test12-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -n -n -n -n test1
+// test12-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -
+// -
+// 2-G-4:~ chuhlig$ echo --
+// --
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo "  -nn "
+//   -nn 
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo "-n test1 -n test2"
+// -n test1 -n test2
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo "test1 -n test2"
+// test1 -n test2
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo ~42
+// ~42
+// 2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -n -nasd
+// -nasd2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -n -n-nnnnn
+// -n-nnnnn2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -nnnnnnn -n -nnn -nnnnn -n-n
+// -n-n2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -nnnnnnn -n -nnn -nnnnn -n feel my pain
+// feel my pain2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo -n -n -n-n
+// -n-n2-G-4:~ chuhlig$ 
+// 2-G-4:~ chuhlig$ echo "'totally logical'"
+// 'totally logical'
+// 2-G-4:~ chuhlig$ echo 'totally logical'
+// totally logical
+// 2-G-4:~ chuhlig$ echo ''totally logical''
+// totally logical
+// 2-G-4:~ chuhlig$ echo ""'totally logical'""
+// totally logical
+
+// 2-G-4:~ chuhlig$ eCho
+
+// 2-G-4:~ chuhlig$ eChO
+
+// 2-G-4:~ chuhlig$ eCHO
+
+// 2-G-4:~ chuhlig$ ECHO
+int	pwd_builtin(void)
+{
+	char	cwd[1028];
+
+	getcwd(cwd, sizeof(cwd));
+	printf("%s\n", cwd);
+	return (0);
+}
```

Comment:
Not sure if this will work how we want later. Depends on how the cd-command works.
If cd changes the return value of getcwd, we need to also set our PWD env var to getcwd.
Else pwd just has to print out what's in our PWD env var, so not too many changes should be needed.